README.md

```diff
@@ -53,8 +104,6 @@ cd build      # enter the build directory
 cmake .. -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX 
 ```
 
-Add `-DNVG_PLUGIN=TRUE` to configure cmake to build nvGraph plugin for cuGraph.
-
 To build the C/C++ code
```

Comment:
I don't see a mention of how to install NVGraph or where the default NVGRAPH_INCLUDE switch should point.

Are those handled by the conda/environments/cugraph_dev.yml setup?

Reply:
NVGraph was added to conda/environments/cugraph_dev.yml in PR #42. However, this PR cannot be merged currently because the NVGraph conda packadge is outdated (in the sense that it conflict with recent cuDF versions in the conda env).